pkg/helm/helm.go

```diff
@@ -149,6 +158,7 @@ func (r *Impl) Install(chart *chart.Chart, kubeNamespace string, releaseName str
 	install := action.NewInstall(cfg)
 	install.ReleaseName = releaseName
 	install.Namespace = kubeNamespace
+	install.Wait = true
```

Comment:
please add Wait to the API as you don't want to mandate waiting. BTW what is the timeout used for wait?

Reply:
https://github.com/IBM/the-mesh-for-data/pull/53
Documentation says that the default timeout is 300 seconds.